testing/classes/TestMethod.lua

```diff
@@ -374,33 +374,55 @@ TestMethod = {
   end,
 
 
+  -- @method - TestMethod:isOS()
+  -- @desc - checks for a specific OS (or list of OSs)
+  -- @param {string||table} - single OS string or list of OSs that are allowed, 
+  --          these will be checked agaisnt love.system.getOS()'s return value
+  -- @return {boolean} - returns true if one of the OSs given matches actual OS
+  isOS = function(self, oss)

```

Comment:
This should use Lua varargs (`...`) instead of a table.

e.g.:

```Lua
isOS = function(self, ...)
  for i = 1, select("#", ...) do
    if love.test.current_os == select(i, ...) then
      return true
    end
  end
  return false
end
```